CHANGES/8404.feature

```diff
@@ -0,0 +1 @@
+Created a manual trigger for our release process so the process could be rerun in case of failures.
```

Comment:
Should this be misc?

Reply:
I was thinking feature since the audience of the changelog is plugin writers and we want to inform them that the release process is now dispatched manually instead of automatically when they push a tag?

Reply:
Right this is the plugin_template. Not pulpcore...